oidc-authservice/tests/test_rock.py

```diff
@@ -38,14 +38,14 @@ def test_rock(rock_test_env):
     check_rock = CheckRock("rockcraft.yaml")
     rock_image = check_rock.get_name()
     rock_version = check_rock.get_version()
-    LOCAL_ROCK_IMAGE = f"{rock_image}:{rock_version}"
+    LOCAL_rock_IMAGE = f"{rock_image}:{rock_version}"
```

Comment:
I dont think we should change the env variables. same comment for changes bellow :)